xtask/src/commands/devtools_test.rs

```diff
@@ -14,6 +14,7 @@ pub(crate) fn run(shell: &Shell) -> Result<(), Box<dyn Error>> {
 
 fn run_tests(shell: &Shell) -> Result<(), Box<dyn Error>> {
     let _r_path = shell.push_dir(R_FOLDER_PATH);
+    cmd!(shell, "Rscript -e rextendr::document()").run()?;
```

Comment:
I ask you kindly to remove this from this PR, and know that we will address this soon, in a separate PR.